### DISCUSSION TITLE
在KTVHTTPCache里面做m3u8格式的支持的话有什么建议？

### DISCUSSION THREAD

**Discussion by fakeyee** [changsanjiang/SJMediaCacheServer#34] [General]
我们项目目前在使用KTVHTTPCache，已经上线运行挺长时间了，但是KTVHTTPCache不支持m3u8格式的视频播放，而且原作者也不再维护了。最近需要支持m3u8格式的视频播放，看到大佬的项目也有部分参考KTVHTTPCache的设计，想请教下大佬，如果在继续使用KTVHTTPCache的情况下，增加m3u8的支持有没有可能实现？本项目的hls部分能否移植过去呢？谢谢

**Comment by changsanjiang** [comment]
不太行。。 处理方式变动有点多，  不太适合移植过去。。。

**Reply by fakeyee** [reply]
谢谢回复，我再研究研究。。
